demo/src/app/components/datepicker/demos/customday/datepicker-customday.ts

```diff
@@ -31,6 +31,6 @@ export class NgbdDatepickerCustomday {
   constructor(private calendar: NgbCalendar) {
   }
 
-  isDisabled = (date: NgbDate, current: {month: number}) => date.month !== current.month;
+  isDisabled = (date: NgbDate, current: {month: number, year: number}) => date.month !== current.month;
```

Comment:
It would be a breaking change, but why not. would be the first date of the month. Initially it was just a `month`, but later there was a feature request, where people asked to add a `year`. In 90% of the cases you would need this to compare `date.month === month`